core/block/editor/file/file.go

```diff
@@ -476,6 +487,24 @@ func (dp *dropFilesProcess) Start(rootId, targetId string, pos model.BlockPositi
 			return
 		}
 		err = cache.Do(dp.picker, smartBlockIds[idx], func(sb File) error {
+			smartBlock, ok := sb.(smartblock.SmartBlock)
```

Comment:
pls don't call everywhere smartblock[idx], flatEntries[idx], I know it was here before, but put them in a separate vars